lighthouse-core/runner.js

```diff
@@ -78,7 +78,7 @@ class Runner {
           // Use canonicalized URL (with trailing slashes and such)
           requestedUrl = new URL(runOpts.url).href;
         } else {
-          throw new LHError(LHError.errors.INVALID_URL);
+          // throw new LHError(LHError.errors.INVALID_URL);
```

Comment:
This is necessary for the callback to work right now. Implement the first part of https://github.com/GoogleChrome/lighthouse/issues/13364 might make dealing with `requestedUrl` easier.